app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;}
+a:visited {color:#000;}
+a:hover   {color:#000;}
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space<br>Opening curly brace `{` should be preceded by one space